chipsec/logger.py

```diff
@@ -140,16 +147,17 @@ def __init__( self ):
         """The Constructor."""
         self.mytime = localtime()
         self.logfile = None
-        self.debug = pyLogging.DEBUG
-        self.info = pyLogging.INFO
         self.rootLogger = pyLogging.getLogger(__name__)
-        self.rootLogger.setLevel(self.debug)
+        self.rootLogger.setLevel(pyLogging.INFO)
         self.ALWAYS_FLUSH = False
-        pyLogging.addLevelName(15, "verbose")
-        self.verbose = 15
+        pyLogging.addLevelName(19, "verbose")
```

Comment:
Would it be useful to have a named color level?

I am thinking in situations where printing decoded in multiple colors helps with the readability; in those scenarios saying print_header() feels awkward when the intention is actually print_blue()

Reply:
The use of print_header and other methods provide additional formatting as well as applying color to the text.  This keeps the log format consistent between modules.  Arbitrary highlighting of text by a module should be avoided.  If this is needed in the long term the log function should be updated to allow for overriding the text color without impacting the consistent text highlighting provided by the other interfaces.